mne/coreg.py

```diff
@@ -1048,56 +1060,58 @@ def scale_mri(subject_from, subject_to, scale, overwrite=False,
             raise IOError("Subject directory for %s already exists: %r"
                           % (subject_to, dest))
 
-    # create empty directory structure
+    logger.debug('create empty directory structure')
     for dirname in paths['dirs']:
         dir_ = dirname.format(subject=subject_to, subjects_dir=subjects_dir)
         os.makedirs(dir_)
 
-    # save MRI scaling parameters
+    logger.debug('save MRI scaling parameters')
     fname = os.path.join(dest, 'MRI scaling parameters.cfg')
     _write_mri_config(fname, subject_from, subject_to, scale)
 
-    # surf files [in mm]
+    logger.debug('surf files [in mm]')
     for fname in paths['surf']:
         src = fname.format(subject=subject_from, subjects_dir=subjects_dir)
         src = os.path.realpath(src)
         dest = fname.format(subject=subject_to, subjects_dir=subjects_dir)
         pts, tri = read_surface(src)
         write_surface(dest, pts * scale, tri)
 
-    # BEM files [in m]
+    logger.debug('BEM files [in m]')
     for bem_name in paths['bem']:
-        scale_bem(subject_to, bem_name, subject_from, scale, subjects_dir)
+        scale_bem(subject_to, bem_name, subject_from, scale, subjects_dir,
+                  verbose=False)
 
-    # fiducials [in m]
+    logger.debug('fiducials [in m]')
```

Comment:
Yeah the comment probably got lost, I just said that the `[in m]` are more code-internal notes and probably not relevant to log messages.

Reply:
`logger.debug` are more or less for internal notes :)

Reply:
It's fine with me :)